icinga2x/Vagrantfile

```diff
@@ -36,7 +37,6 @@ nodes = { 'icinga2' => {
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   nodes.each_pair do |name, options|
     config.vm.define name do |node_config|
-      node_config.vm.box = options[:box_virtualbox]
```

Comment:
Ah. I could just remove the vm.box for openstack and it'll go through.
I'll amend and add this back for virtualbox.